build_tools/circle/install.sh

```diff
@@ -32,7 +32,7 @@ popd
 conda create -n testenv --yes python pip pytest nose
 source activate testenv
 
-pip install -e.
+pip install -e .[plots]
```

Comment:
This doesn't work, needs single quotes:

`pip install -e '.[plots]'`

Reply:
👍